apps/judicial-system/web/src/components/CaseFileList/CaseFileList.tsx

```diff
@@ -44,17 +46,19 @@ const CaseFileList: React.FC<Props> = (props) => {
               file={file as TCaseFile}
               showFileSize={true}
               defaultBackgroundColor={
-                file.state === CaseFileState.BOKEN_LINK ? 'dark100' : 'blue100'
+                file.status === 'broken' ? 'dark100' : 'blue100'
```

Comment:
I was a bit confused reading this. First I read `file.status` as `file.state` and I did not know where this 'broken' was defined. Then I figured out these were two different properties of the file.

I want to suggest that we rename the `file.state` to `file.storage` with the options `IN_COURT | IN_RVG | DELETED`
we can keep the `file.status` as it is.

Just a suggestion 🤓

Reply:
I agree, this is somewhat confusing. However, renaming the `state` field is a rather big change, it comes all the way from the database. On the other hand `status` is defined in the client and could be easily changed to something different, for instance `uploadStatus`. I will create a task in the Asana inbox for refactoring this.